COPYING

```diff
@@ -14,11 +14,11 @@
       the copyright owner that is granting the License.
 
       "Legal Entity" shall mean the union of the acting entity and all
-      other entities that control, are controlled by, or are under common
```

Comment:
Did you notice that this file is the [Apache License](https://en.wikipedia.org/wiki/Apache_License). Did you check with a laywer that this is still the same license? If the license text is changed, the license needs to be approved again by the "open source initiative".

Reply:
Agree with @koppor , this changes MUST be reverted too.